modules/mobile/spec/request/claims_document_upload_spec.rb

```diff
@@ -41,53 +41,54 @@
   end
 
-  # It was found that FE is using both different casing between multi image upload and single image upload.
-  context 'when snake case is used for parameters' do
+  context 'when camel case is used for parameters and camel case header is disabled' do
     it 'uploads a file' do
-      params = { file:, claim_id:, tracked_item_id:, document_type: }
+      params = { file:, claimId: claim_id, trackedItemId: tracked_item_id, documentType: document_type }
       expect do
-        post '/mobile/v0/claim/600117255/documents', params:, headers: sis_headers
+        post '/mobile/v0/claim/600117255/documents', params:, headers: sis_headers(camelize: false)
       end.to change(Lighthouse::DocumentUpload.jobs, :size).by(1)
 
       expect(response.status).to eq(202)
-      expect(response.parsed_body.dig('data', 'jobId')).to eq(Lighthouse::DocumentUpload.jobs.first['jid'])
+      expect(response.parsed_body.dig('data', 'job_id')).to eq(Lighthouse::DocumentUpload.jobs.first['jid'])
       expect(Lighthouse::DocumentUpload.jobs.first.dig('args', 1, 'tracked_item_id')).to eq([tracked_item_id])
     end
 
     it 'uploads multiple jpeg files' do
       files = [Base64.encode64(File.read('spec/fixtures/files/doctors-note.jpg')),
                Base64.encode64(File.read('spec/fixtures/files/marriage-cert.jpg'))]
-      params = { files:, claim_id:, tracked_item_id:, document_type: }
+      params = { files:, claimId: claim_id, trackedItemId: tracked_item_id, documentType: document_type }
       expect_any_instance_of(BenefitsDocuments::Service).to receive(:cleanup_after_upload)
       expect do
         post '/mobile/v0/claim/600117255/documents/multi-image', params: params.to_json,
-                                                                 headers: sis_headers(json_body_headers)
+                                                                 headers: sis_headers(camelize: false, json: true)
       end.to change(Lighthouse::DocumentUpload.jobs, :size).by(1)
+
       expect(response.status).to eq(202)
-      expect(response.parsed_body.dig('data', 'jobId')).to eq(Lighthouse::DocumentUpload.jobs.first['jid'])
+      expect(response.parsed_body.dig('data', 'job_id')).to eq(Lighthouse::DocumentUpload.jobs.first['jid'])
       expect(Lighthouse::DocumentUpload.jobs.first.dig('args', 1, 'tracked_item_id')).to eq([tracked_item_id])
     end
   end
 
   it 'uploads multiple gif files' do
     files = [Base64.encode64(File.read('spec/fixtures/files/doctors-note.gif')),
              Base64.encode64(File.read('spec/fixtures/files/marriage-cert.gif'))]
-    params = { files:, claimId: claim_id, trackedItemId: tracked_item_id, documentType: document_type }
+    params = { files:, claim_id:, tracked_item_id:, document_type: }
     expect_any_instance_of(BenefitsDocuments::Service).to receive(:cleanup_after_upload)
+
     expect do
       post '/mobile/v0/claim/600117255/documents/multi-image', params: params.to_json,
```

Comment:
I guess you could make an argument that we should for the multi-file upload this shouldn't be json to better match reality. But... I also don't think we should be too concerned about that. The front end could change how they do things at any moment, so it's not our role to match how they use the api. We just want to ensure that the code works with both camel and snake.